main/config.c

```diff
@@ -45,6 +45,10 @@
 #include <time.h>
 #include <sys/stat.h>
 
+#if defined(__FreeBSD__)
```

Comment:
I don’t _think_ this `#include` needs to be guarded by the `#if`. Both the POSIX docs and the Linux manpage specifically mention `sys/wait.h`.